public/static/docs/command-reference/metrics/modify.md

```diff
@@ -33,22 +33,22 @@ ERROR: failed to modify metric file settings -
 
 ## Options
 
-- `-t`, `--type` - specify a type of the metric file. Accepted values are:
-  `raw`, `json`, `tsv`, `htsv`, `csv`, `hcsv`. It will be saved into the
+- `-t`, `--type` - specify a type of the metric file. Accepted values are: `raw`
+  (default), `json`, `tsv`, `htsv`, `csv`, `hcsv`. It will be saved into the
```

Comment:
not sure I understand what does `default` mean here?

Reply:
It means that `raw` is the default `--type`. Looks like this when rendered:
![image](https://user-images.githubusercontent.com/1477535/74303662-40d1b980-4d20-11ea-96d1-96130e0179b5.png)

Reply:
the question is - do we actually put the `raw` type into DVC-file by default?

Reply:
Yes, with or without `-t raw` we do:
![image](https://user-images.githubusercontent.com/1477535/74391727-ab3f3400-4dca-11ea-9387-ad27733d582a.png)

Reply:
I just trying to remember how does it actually work with DVC figuring out the type by the extension .. if raw is specified on the command line or in the DVC-file - will it still try to do extension analysis? can I omit the type in the DVC-file?

Reply:
So you mean to `dvc metrics show`? Honestly I'm not sure, I haven't played much with metrics add/show -t or -x options. I just tried to metrics add a `m.json` file and it didn't autodetect `json` so maybe this whole module needs to be checked separate to this PR? Maybe it has some bugs or if not it an be much better explained in docs

Reply:
I hope that it was explained right :) It's hard to check for changes w/o understanding the semantics. If you are sure that we don't change semantics it's fine.